### PR TITLE
Change visual uses of Paypal to PayPal

### PR DESCRIPTION
**Pull request by lukespragg**

### CODE REVIEW

File: app/models/payment_notification/paypal.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@
 #  index_payment_notifications_on_type      (type)
 #
 
-class PaymentNotification::Paypal < PaymentNotification
+class PaymentNotification::PayPal < PaymentNotification
```
Comment (juanj) [inline app/models/payment_notification/paypal.rb]: This is breaking the code. You should not change it on the code or, if you are going change it, change it every where.

File: app/models/payment_notification/paypal.rb [hunk 2]
```diff
@@ -80,20 +80,20 @@ def verified?
   end
 
   def checkout_method
-    Account::Paypal.instance
+    Account::PayPal.instance
```
Comment (juanj) [inline app/models/payment_notification/paypal.rb]: Same for this

File: app/models/payment_notification/paypal.rb [hunk 3]
```diff
@@ -80,20 +80,20 @@ def verified?
...
   end
 
   # find or create the order
   def process_order
-    Transaction::Order::Paypal.find_by_id(shopping_cart.order_id) || Transaction::Order::Paypal.create_from_payment_notification(notification_id: self.id)
+    Transaction::Order::PayPal.find_by_id(shopping_cart.order_id) || Transaction::Order::PayPal.create_from_payment_notification(notification_id: self.id)
```
Comment (juanj) [inline app/models/payment_notification/paypal.rb]: This

File: app/models/payment_notification/paypal.rb [hunk 4]
```diff
@@ -80,20 +80,20 @@ def verified?
...
   end
 
   private
 
-  # POST back to Paypal to protect against spoofing
+  # POST back to PayPal to protect against spoofing
   def post_back(raw_post)
     if Rails.env.test?
-      raise "PaymentNotification::Paypal(#{id}) trying to POST back verify in test. Stub out :post_back to 'VERIFIED' for successful verification"
+      raise "PaymentNotification::PayPal(#{id}) trying to POST back verify in test. Stub out :post_back to 'VERIFIED' for successful verification"
```
Comment (juanj) [inline app/models/payment_notification/paypal.rb]: And this.

File: app/models/payment_notification/paypal.rb (hunk 1 above)
Comment (lukespragg) [inline app/models/payment_notification/paypal.rb]: This PR was only for the visual, didn't want to break it with the other changes.

**Comment by rappo** [comment]
@lukespragg @juanj if either of you want to update this so that only the front end -- and no code -- is affected, I'll merge it in.

**Pull request closed without merging**